pkg/systemd/services/service-details.jsx

```diff
@@ -561,7 +561,7 @@ export class ServiceDetails extends React.Component {
                                 <hr />
                                 {relationships.map(rel =>
                                     rel.Units && rel.Units.length > 0 &&
-                                        <DescriptionListGroup key={rel.Name.split().join("")}>
+                                        <DescriptionListGroup key={rel.Name.split(" ").join("")}>
```

Comment:
but... why? What is wrong with `key={rel.Name}`? The spaces don't hurt at all in a key. And as split() is a no-op here, it obviously worked before :grin: